tests/e2e/csi_static_provisioning_file_basic.go

```diff
@@ -0,0 +1,239 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"context"
+	"path/filepath"
+	"time"
+
+	"sigs.k8s.io/vsphere-csi-driver/pkg/csi/service/common"
```

Comment:
Can you fix the ordering of imports? Its normally:

```
<inbuilt libraries>

<external libraries>

<internal libraries> 
```

Reply:
Done